### PR TITLE
Add a PR template

### PR DESCRIPTION
**Pull request by aliddell**
Suggestions welcome. Should go in all the repos once it's approved.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+## Type of change
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Documentation
+- [ ] Other (please describe):
+
+# Changes
+
```
Comment (andy-sweet) [inline .github/pull_request_template.md:20]: I assume that the format here follows the same style/format as the changelog. In that case, I think we should add an instruction to indicate that, maybe with the same link to `Keep a changelog`.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
+# Description
+
+*Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.*
+
+## Related Issue(s)
```
Comment (andy-sweet) [inline .github/pull_request_template.md:5]: Nit: other section titles use Sentence case, whereas this uses Title Case.

```suggestion
## Related issue(s)
```

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+-
+
+# How has this been tested?
+
+*Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.*
+
+# Checklist
+
+- [ ] My code follows the style guidelines of this project.
```
Comment (andy-sweet) [inline .github/pull_request_template.md:51]: Where are the style guidelines?

**Review by andy-sweet — APPROVED** [review]
As discussed in standup, this is good start, definitely better than the nothing we have now, and we can iteratively improve this as needed.

I made a few comments, some of which might be worth addressing, but none of which are blocking.

File: .github/pull_request_template.md (hunk 3 above)
Comment (aliddell) [reply .github/pull_request_template.md]: That's the neat part. There are none. (Fixed.)

**Comment by aliddell** [comment]
On closer consideration, you @andy-sweet and @nclack were right that putting changes here in changelog format are absolutely duplication of effort. If the submitter has checked all the boxes, the reviewer should be able to tell from looking at the changelog, if nothing else, what's changed, and in that exact format.

**Review by nclack — APPROVED** [review]

**Pull request merged**